server/routes/ssot.js

```diff
@@ -3,8 +3,8 @@ const express = require('express');
 const router = express.Router();
 
 // Require controller modules
-const parsingController = require('../controllers/ssotParsingController');
 const retrievalController = require('../controllers/ssotRetrievalController');
+const parsingController = require('../controllers/ssotParsingController');
```

Comment:
this change was not really necessary 😄

Reply:
You don't need to change it back, just keep an eye on it 😉

Reply:
I think that change comes from merging ;) But I agree it is unnecessary